### ISSUE TITLE
Question about SceneFlow Model and Fine-tuning on KITTI Dataset

### ISSUE THREAD

**Issue by laycc** [jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching#2]
Dear jiaxiZeng,

I hope this message finds you well. I am currently working on reproducing your code, and I encountered an issue. I used the SceneFlow weight file provided by you, specifically the "pcvnet_sceneflow_sigma32.pth" file. Upon fine-tuning this weight file on the KITTI dataset, I obtained a D1 result of 3.476 on the KITTI validation set, which is significantly different from the result you provided (1.93). I'm wondering if I made a mistake somewhere or if there are specific considerations to be mindful of during the fine-tuning process.

Additionally, could you please share the fine-tuned weight file for KITTI (.pth) if it's convenient for you? I would greatly appreciate your assistance.

Thank you very much for your response.

Best regards

Below are experimental attachments：
![param](https://github.com/jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching/assets/41206490/28b39c62-7828-466c-998d-53adba0c3e48)
![exp](https://github.com/jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching/assets/41206490/a17da4bd-e5de-460c-bda1-fa40dad32039)

**Comment by jiaxiZeng** [comment]
Hi,

The checkpoint of kitti submission is [here](https://www.dropbox.com/scl/fi/up40dfekhs1mhs67po3ti/pcvnet_kitti_sigma32.pth?rlkey=4ic34nt71sz3b2d9xykcwk93a&dl=0).

You might check your hyperparameters, e.g., learning rate or batchsize, when finetuning on the kitti dataset.

Best wishes.

**Comment by laycc** [comment]
Dear jiaxi,

Thank you very much for your prompt assistance. I tested the KITTI validation set using the weight file you provided ("pcvnet_kitti_sigma32.pth"), and the result was 1.13. This further indicates that there might be errors in my code reproduction process. Despite fine-tuning with the hyperparameters from your paper, the performance remains unsatisfactory. I'm uncertain where the issue lies. Could you please review my hyperparameters to see if they are reasonable, or alternatively, could you share the hyperparameters you used for fine-tuning on the KITTI dataset? Your help is greatly appreciated.

Thank you again for your support.

Best regards.

Below are experimental attachments：
![train](https://github.com/jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching/assets/41206490/20707ed0-c10f-44d4-a0f4-b84c740440cc)
![train1](https://github.com/jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching/assets/41206490/b4a5fae4-a0c9-4549-a1c2-e7409ef9d855)

**Comment by jiaxiZeng** [comment]
Hi，
the following is the training parameters of KITTI2015&2012 dataset.

`train_stereo.py --mixed_precision --batch_size 12 --train_datasets kitti --lr 0.00001 --num_steps 10000 --image_size 320 1024 --train_iters 6 --valid_iters 4 --valid_freq 100 --device 0 1 --saving_path ./pcv_ckpts --max_disp 512 --slow_fast_gru --saturation_range 0.7 1.3 --spatial_scale -0.2 0.2 --restore_ckpt ./pcv_ckpts/pcvnet_sceneflow_sigma32.pth --init_sigma 32`

You can try again or you should check your datasets for errors.

Best wishes to you.

**Comment by laycc** [comment]
Dear jiaxi,

Thank you once again for your invaluable assistance. I successfully reproduced similar results using the hyperparameters you provided, which further highlights the issues with my previous set of hyperparameters.

I sincerely appreciate your help and wish you a pleasant and joyful life.

Best regards.

Below are experimental attachments（Verify on 200 training dataset）：
The test results of the weight file you provided：
![1](https://github.com/jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching/assets/41206490/3cf2a2d6-e14f-4279-93e2-79a34c8e9e83)
My reproduction results：
![image](https://github.com/jiaxiZeng/Parameterized-Cost-Volume-for-Stereo-Matching/assets/41206490/8544ff14-4dd3-4203-a0fe-0122261ba8a6)
